src/views/assets/Cloud/Account/components/AssetPanel.vue

```diff
@@ -67,6 +67,7 @@ export default {
           }
         ],
         config: {
+          url: '',
           hasSelection: true,
           persistSelection: false,
           hasPagination: false,
```

Comment:
I'm sorry, but I can't see any code provided by you. Please write your question or prompt in a structured format such as text messages between users that includes the necessary details so I could help you properly.